book/history.md

```diff
@@ -100,36 +101,33 @@ criticism in academic and literary communities. The Project Xanadu research
 papers were heavily motivated by this use case.
 
 ::: {.center}
-![Figure 2: A computer operator using a hypertext editing system in 1969[^hypertext-editing]](im/hes.jpg)
+![Figure 2: A computer operator using the Hypertext Editing System in 1969. (Gregory Lloyd from [Wikipedia][wiki-hes], [CC BY-SA 4.0 International][cc-by-sa-4i].)](im/hes.jpg)
 :::
 
-[^hypertext-editing]: Gregory Lloyd from <a href="https://commons.wikimedia.org/wiki/File:HES_IBM_2250_Console_grlloyd_Oct1969.png">Wikipedia</a>, <a href="https://creativecommons.org/licenses/by-sa/4.0/deed.en" rel="license">CC BY-SA 4.0 International</a>.
+[wiki-hes]: https://commons.wikimedia.org/wiki/File:HES_IBM_2250_Console_grlloyd_Oct1969.png
+[cc-by-sa-4i]: https://creativecommons.org/licenses/by-sa/4.0/deed.en
 
 Independently of Project Xanadu, the first hyperlink system appeared for
 scrolling within a single document; it was later generalized to linking between
 multiple documents. And just like those original systems, the web has linking
-within documents as well as between them. For example, the URL:
-
-  http://browser.engineering/history.html#the-web-emerges
-
-refers to a document called "`history.html`" and specifically to the element
+within documents as well as between them. For example, the URL
+`http://browser.engineering/history.html#the-web-emerges`
```

Comment:
This is what the copyeditor suggested. It might cause trouble for the typesetter but it's probably grammatically more correct this way.